generators/app/templates/src/main/java/package/web/rest/EntityKafkaResource.java.ejs

```diff
@@ -0,0 +1,65 @@
+<%#
+ Copyright 2013-2020 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+package <%= packageName %>.web.rest;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.*;
+
+import <%= packageName %>.service.kafka.producer.<%= entityClass %>Producer;
+<%_ if (type !== 'String') { _%>
+import <%= packageName %>.domain.<%= entityClass %>;
+<%_ } _%>
+
+/**
+<%_ if (type !== 'String') { _%>
+ * REST controller for managing {@link <%= packageName %>.domain.<%= entityClass %>} through Kafka.
+<%_ } else { _%>
+ * REST controller for managing String through Kafka.
+<%_ } _%>
+ */
+@RestController
+@RequestMapping("/api")
+@Transactional
+public class <%= entityClass %>KafkaResource {
+
+    private final Logger log = LoggerFactory.getLogger(<%= entityClass %>KafkaResource.class);
+
+    private final <%= entityClass %>Producer <%= camelCaseEntityClass %>Producer;
+
+    public <%= entityClass %>KafkaResource(<%= entityClass %>Producer <%= camelCaseEntityClass %>Producer) {
+        this.<%= camelCaseEntityClass %>Producer = <%= camelCaseEntityClass %>Producer;
+    }
+
+    /**
+     * {@code POST  /<%= camelCaseEntityClass %>s/kafka} : Send a <%= camelCaseEntityClass %> in Kafka.
+     *
+<%_ if (type !== 'String') { _%>
+     * @param <%= camelCaseEntityClass %> the <%= camelCaseEntityClass %> to send.
+<%_ } else { _%>
+     * @param <%= camelCaseEntityClass %> the String to send.
+<%_ } _%>
+     */
+    @PostMapping("/<%= camelCaseEntityClass %>s/kafka")
```

Comment:
```suggestion
    @PostMapping("/<%= camelCaseEntityClass %>s/produce")
```
or 
```suggestion
    @PostMapping("/<%= camelCaseEntityClass %>s/kafka/produce")
```
it could be more revealing about the action

Reply:
As the POST verb is used, we already know we are producing something.